src/components/Committee.tsx

```diff
@@ -466,7 +466,7 @@ export default class Committee extends React.Component<Props, State> {
           <CommitteeShareHint committeeID={this.props.match.params.committeeID} />
         </Segment>
         <Segment>
-          <Label attached="top left" size="large">Keyboard Shorcuts</Label>
+          <Label attached="top left" size="large">Keyboard shorcuts</Label>
```

Comment:
Don't take the shortcut, put a 't' in here :p

Reply:
Good catch